pkgs/applications/networking/mailreaders/afew/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, pythonPackages, notmuch }:
+
+let
+  pname = "afew";
+  version = "1.2.0";
+in
+
+pythonPackages.buildPythonApplication {
+  inherit pname version;
+
+  src = pythonPackages.fetchPypi {
+    inherit pname version;
+    sha256 = "121w7bd53xyibllxxbfykjj76n81kn1vgjqd22izyh67y8qyyk5r";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [
+    setuptools_scm pythonPackages.notmuch chardet
+  ] ++ stdenv.lib.optional (!pythonPackages.isPy3k) subprocess32;
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = "${version}";
+
+  postInstall = ''
+    wrapProgram $out/bin/afew \
+      --prefix LD_LIBRARY_PATH : ${notmuch}/lib
```

Comment:
Can this be fixed by hardcoring the path to the lib. Using LD_LIBRARY_PATH should be prevented

Reply:
It's already hardcoded into `pythonPackages.notmuch`. Will strip this.